project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info

```diff
@@ -30,12 +30,12 @@ features[flag][] = subscribe_c4m_follow_node
 features[flag][] = subscribe_c4m_follow_term
 features[og_features_permission][] = node:group:invite any user
 features[og_features_permission][] = node:project:invite any user
+features[user_permission][] = flag subscribe_c4m_follow_comment
```

Comment:
@ordavidil 
I recreated the feature. It changed the order to alphabetic.
That's why we got this switch.
We'll have to revert it every time feature is created. Why not change it to default?